From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Guobing Chen <guobing.chen@intel.com>
Date: Wed, 3 Apr 2019 13:47:39 +0800
Subject: [PATCH] Add option for pgo profiling test with scimark

Signed-off-by: Guobing Chen <guobing.chen@intel.com>

-------------------------------------------------------------------
Refactored patch: 0003-Add-option-for-pgo-profiling-test-with-scimark.patch
Signed-off-by: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
--------------------------------------------------------------------
---
 Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f398ec4..cf18b43 100644
--- a/Makefile
+++ b/Makefile
@@ -58,6 +58,9 @@ $(PLATS) help test clean:
 test:	dummy
 	src/lua -v
 
+test_pgo:      dummy
+	src/lua scimark.lua
+
 install: dummy
 	cd src && $(MKDIR) $(INSTALL_BIN) $(INSTALL_INC) $(INSTALL_LIB) $(INSTALL_MAN) $(INSTALL_LMOD) $(INSTALL_CMOD)
 	cd src && $(INSTALL_EXEC) $(TO_BIN) $(INSTALL_BIN)
@@ -112,6 +115,6 @@ pc:
 	@echo "includedir=$(INSTALL_INC)"
 
 # list targets that do not create files (but not all makes understand .PHONY)
-.PHONY: all $(PLATS) clean test install local none dummy echo pecho lecho pc
+.PHONY: all $(PLATS) clean test test_pgo install local none dummy echo pecho lecho pc
 
 # (end of Makefile)
